api-admin/src/main/java/bio/terra/pearl/api/admin/controller/logging/LoggingController.java

```diff
@@ -0,0 +1,48 @@
+package bio.terra.pearl.api.admin.controller.logging;
+
+import bio.terra.pearl.api.admin.api.LoggingApi;
+import bio.terra.pearl.api.admin.service.auth.AuthUtilService;
+import bio.terra.pearl.api.admin.service.logging.LoggingExtService;
+import bio.terra.pearl.core.model.admin.AdminUser;
+import bio.terra.pearl.core.model.log.LogEventType;
+import bio.terra.pearl.core.service.exception.PermissionDeniedException;
+import jakarta.servlet.http.HttpServletRequest;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class LoggingController implements LoggingApi {
+
+  private final LoggingExtService loggingExtService;
+  private final AuthUtilService authUtilService;
+  private final HttpServletRequest request;
+
+  public LoggingController(
+      LoggingExtService loggingExtService,
+      AuthUtilService authUtilService,
+      HttpServletRequest request) {
+    this.loggingExtService = loggingExtService;
+    this.authUtilService = authUtilService;
+    this.request = request;
+  }
+
+  @Override
+  public ResponseEntity<Object> get(String days, String eventTypes) {
+    AdminUser operator = authUtilService.requireAdminUser(request);
+
+    if (!operator.isSuperuser()) {
+      throw new PermissionDeniedException("You do not have permission for this operation");
+    }
+
+    if (StringUtils.isBlank(eventTypes)) {
```

Comment:
this feels like it's blending parse and business logic.  Let's parse the evertTypes string into a list, and then let the ExtService decide what to do with the (possibly empty) list